src/main/java/org/spongepowered/mod/mixin/entity/MixinEntity.java

```diff
@@ -90,6 +92,13 @@
     public abstract void mountEntity(net.minecraft.entity.Entity entityIn);
 
     @Override
+    public UUID getUniqueId() {
+        Entity spongeEntity = (Entity)this;
```

Comment:
The `getUniqueID` method is in NMS. Using `this.getUniqueID()` isn't possible because it's not implemented in this class or any superclass it inherits. The weird casting is based on this convention further down: https://github.com/SpongePowered/Sponge/blob/52d67d0b757950b09d489415928eaa8b002944c4/src/main/java/org/spongepowered/mod/mixin/entity/MixinEntity.java#L108-109